src/scss/layout/_anchor-link.scss

```diff
@@ -0,0 +1,59 @@
+@use '~accoutrement/sass/tools';
+
+.anchor-link-wrapper {
+  display: inline-block;
+  position: relative;
+
+  h2 {
+    display: inline-block;
+    padding-left: var(--anchor-link-padding-l, var(--gutter));
+
+    @include tools.above('wide-page') {
+      --anchor-link-padding-l: 0;
+    }
+  }
+}
+
+.header-anchor {
+  position: absolute;
+  left: var(--anchor-position-offset-l, -0.25em);
+  top: var(--anchor-position-offset-t, var(--gutter-plus));
+  transform-origin: top center;
+  width: var(--gutter);
+
+  @include tools.above('wide-page') {
+    --anchor-position-offset-l: -1em;
+    --anchor-position-offset-t: 1.25em;
+  }
+
+  &.anchor-clicked {
+    @include tools.transition('move');
+
+    svg {
+      transform: rotate(-45deg);
+    }
+  }
+
+  &:visited {
+    --fill: var(--active);
+  }
+}
+
+.anchor-clicked::before {
+  background: var(--feature);
```

Comment:
I would add an animation in and out on the text that is shown when copied. ... something like 

``` 
@keyframes fadein-out {
  10% {
    opacity: 1;
  }

  90% {
    opacity: 1;
  }
}

.anchor-clicked::before {
animation: fadein-out 3000ms forwards;
...
opacity: 0; // so it is not shown right away
```

Also we typically will nest the `::before`. It helps if you are adding other styles to `.anchor-clicked` like:

```
.anchor-clicked {
  &::before {
  ...
  }
  
  svg {
    transform: rotate(-45deg);
  }
}  
```

Reply:
good catch. I also added an animation to the `link copied` text